src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,38 @@
+@import '../utils/variables';
+
+.card__rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+.reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: $card-text-color;
```

Comment:
The variable `$card-text-color` is used here but is not defined. Ensure it is defined or imported correctly.